test/docker_tests/test_container_environment.py

```
@@ -0,0 +1,950 @@
+    )
+    # Combine and clean stdout and stderr
+    stdouterr = (
+        re.sub(r'\x1b\[[0-9;]*m', '', result.stdout or '') +
+        re.sub(r'\x1b\[[0-9;]*m', '', result.stderr or '')
+    )   
+    result.output = stdouterr
+    # Print container output for debugging in every test run.
+    try:
+        print("\n--- CONTAINER out ---\n", result.output)
+    except Exception:
+        pass
```

Ruff (0.14.1)

`try`-`except`-`pass` detected, consider logging the exception

(S110)